core/init_test.go

```diff
@@ -0,0 +1,16 @@
+package core
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsInit(t *testing.T) {
```

Comment:
Thanks for pointing that. I think that is right to have a test here but I am not sure that the time it would take to simulate errors coming from OS FileSystem level justify the time. Can you pls make a list to collect all the places where you think tests are missing? So we can return to it and fulfill the techdebt.